cartography/intel/gcp/__init__.py

```diff
@@ -1,11 +1,35 @@
 from oauth2client.client import GoogleCredentials, ApplicationDefaultCredentialsError
+import googleapiclient.discovery
 import logging
 
-from cartography.intel.gcp import crm
+from cartography.intel.gcp import crm, compute
 
 logger = logging.getLogger(__name__)
 
 
+def _get_crm_resource_v1(credentials):
```

Comment:
We may want to be consistent on documenting parameters

Reply:
Fair point, I can add these.